### DISCUSSION TITLE
Excluding IP address from being detected by suricata and p0f

### DISCUSSION THREAD

**Discussion by gonresendes** [telekom-security/tpotce#1697] [Q&A]
Is it possible to exclude IP addresses from being detected by suricata and p0f?
I don't want these IPs to stop communicating with TPOT, otherwise I'd try using iptables, but I just don't want to be alerted by these IP addresses.
Any ideas?

**Comment by t3chn0m4g3** [comment]
1. Follow the Wiki for using Docker volumes.
2. Adjust the [Suricata Capture Filter](https://github.com/telekom-security/tpotce/blob/24.04.1/docker/suricata/dist/capture-filter.bpf) and copy it i.e. to `$HOME/tpotce/data/suricata`.
3. Follow the Suricata `docker/Suricata/Dockerfile` for the correct volume location and adjust the Suricata Service in the `$HOME/tpotce/docker-compose.yml` by adding a new volume location.

**Comment by gonresendes** [comment]
And P0f?
